Makefile

```diff
@@ -5,7 +5,13 @@ obj-m += cr14.o
 dtbo-y += cr14.dtbo
 
 targets += $(dtbo-y)
-always  := $(dtbo-y)
+
+# Gracefully supporting the new always-y without cutting off older target with kernel 4.x
+ifeq ($(firstword $(subst ., ,$(KERNELRELEASE))),4)
+	always-y := $(dtbo-y)
```

Comment:
It's probably the other way around. `always` for kernel 4.x and `always-y` for newer kernels.